budgetportal/templates/components/about/Media/index.html

```diff
@@ -0,0 +1,62 @@
+<div id="media-and-other-information">
```

Comment:
it looks like these are copied in `assets/js/components/about/Media/index.html` as well - can you clean up the ones in `assets/js`?

Reply:
that's why their diffs are showing up

Reply:
Fixed in step 3 PR